net.crossfireatlas.crossfire.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.crossfireatlas.crossfire</id>
```

Comment:
there doesn't seem to be a working `crossfireatlas.net` and no mention of it here.

App-id as supposed to reference a domain controlled by the developers.

Reply:
There is crossfireatlas.net and it works (just tested):
http://www.crossfireatlas.net/
So, I am not sure what the problem is?

Reply:
Not sure what happened to my response. Anyway:
crossfireatlas.net exists. You can check:
http://www.crossfireatlas.net/
So, not sure what the problem is?

Reply:
no problem. it didn't seem to work when I checked.

Reply:
We decided (per discussion on the crossfire developer mailing list) to use crossfireatlas.net rather than crossfire.real-time.com because of the dash (flatpak does not seem to like dashes).